action.yml

```diff
@@ -24,6 +24,11 @@ inputs:
     description: 'Comma-separated list of file patterns to exclude'
     required: false
     default: ''
+  HUMAN_LANGUAGE:
+    description: 'The human language to use for code review'
+    required: false
+    default: 'en'
```

Comment:
While adding support for different human languages is a good feature, the `HUMAN_LANGUAGE` input lacks validation.  Consider adding validation to ensure that only supported languages are used to prevent unexpected behavior or errors.  For instance, a simple check against a whitelist of supported languages could be implemented. This will improve the robustness of the action.